src/main/flight/pid.c

```diff
@@ -399,6 +399,8 @@ static FAST_RAM_ZERO_INIT float acErrorLimit;
 
 void pidResetITerm(void)
 {
+    extern float integratedYaw;
+    integratedYaw *= 0.99f;
```

Comment:
Why do you want yaw to slowly unwind in here?

Reply:
I don't want yaw to go crazy while hand starting

Reply:
Why not just setting it to zero like the rest?